drop-in/src/main/java/com/adyen/checkout/dropin/ui/giftcard/GiftCardPaymentConfirmationDialogFragment.kt

```diff
@@ -95,10 +96,8 @@ internal class GiftCardPaymentConfirmationDialogFragment : DropInBottomSheetDial
 
         val paymentMethods = alreadyPaidMethods + currentPaymentMethod
 
-        val imageLoader = ImageLoader.getInstance(
-            requireContext(),
-            dropInViewModel.dropInConfiguration.environment
-        )
+        val imageLoader = DefaultImageLoader.with(dropInViewModel.dropInConfiguration.environment)
```

Comment:
I know this was already here but getting value directly from view model is not best practices. we should find a way to use observers.

Reply:
I guess this might needs another PR because `dropInViewModel.dropInConfiguration` is used in a lot of places not just here.